ui/StatusQ/src/StatusQ/Components/StatusChatList.qml

```diff
@@ -40,238 +43,164 @@ Item {
-                StatusChatListCategoryItem {
-                    id: statusChatListCategoryItem
-                    objectName: "categoryItem"
-                    
-                    function setupPopup() {
-                        categoryPopupMenuSlot.item.categoryItem = model
-                    }
-
-                    Connections {
-                        enabled: categoryPopupMenuSlot.active && statusChatListCategoryItem.highlighted
-                        target: categoryPopupMenuSlot.item
-                        function onClosed() {
-                            statusChatListCategoryItem.highlighted = false
-                            statusChatListCategoryItem.menuButton.highlighted = false
-                        }
-                    }
-
-                    title: model.name
 
-                    opened: model.categoryOpened
-
-                    sensor.pressAndHoldInterval: 150
-                    propagateTitleClicks: true // title click is handled as a normal click (fallthru)
-                    showAddButton: showCategoryActionButtons
-                    showMenuButton: !!root.onPopupMenuChanged
-                    highlighted: false//statusChatListCategory.dragged // FIXME DND
-                    
-                    hasUnreadMessages: model.hasUnreadMessages
-                    
-                    onClicked: {
-                        if (!sensor.enabled) {
-                            return
-                        }
-                        if (mouse.button === Qt.RightButton && showCategoryActionButtons && !!root.categoryPopupMenu) {
-                            statusChatListCategoryItem.setupPopup()
-                            highlighted = true;
-                            categoryPopupMenuSlot.item.popup()
-                        } else if (mouse.button === Qt.LeftButton) {
-                            root.model.sourceModel.changeCategoryOpened(model.categoryId, !statusChatListCategoryItem.opened)
-                        }
-                    }
-                    onToggleButtonClicked: root.model.sourceModel.changeCategoryOpened(model.categoryId, !statusChatListCategoryItem.opened)
-                    onMenuButtonClicked: {
-                        statusChatListCategoryItem.setupPopup()
-                        highlighted = true
-                        menuButton.highlighted = true
-                        categoryPopupMenuSlot.item.popup()
-                    }
-                    onAddButtonClicked: {
-                        root.categoryAddButtonClicked(categoryId)
+            StatusDraggableListItem {
+                id: draggableItem
+                width: parent.width
+                height: visible ? implicitHeight : 0
+                dragParent: statusChatListItems
+                visualIndex: chatLoader.visualIndex
+                draggable: (statusChatListItems.count > 1)
+                horizontalPadding:0
+                verticalPadding: 0
```

Comment:
Just `padding` should do

Reply:
it doesn't, unfortunately:
with just padding: 0
<img width="307" alt="padding" src="https://user-images.githubusercontent.com/31625338/229158214-304e5648-3c83-4968-a725-b8266d0e3858.png">

with horizontal & vertical padding: 0
<img width="305" alt="horizontalPadding_verticalPadding" src="https://user-images.githubusercontent.com/31625338/229158275-4e063e62-ce06-4425-9573-1b125c65989b.png">